benchmarks/performance-poetry/complex-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+plugins {
+  id("com.android.test")
+  id("org.jetbrains.kotlin.android")
+}
+
+// Note: We are not including our defaults from .buildscript as we do not need the base Workflow
+// dependencies that those include.
+
+android {
+  compileSdk = 32
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
+
+  defaultConfig {
+    minSdk = 26
+    targetSdk = 32
+
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+  }
+
+  buildTypes {
+    // This benchmark buildType is used for benchmarking, and should function like your
+    // release build (for example, with minification on). It"s signed with a debug key
+    // for easy local/CI testing.
+    create("benchmark") {
+      isDebuggable = true
+      signingConfig = getByName("debug").signingConfig
+      matchingFallbacks.add("release")
+      proguardFile("baseline-proguard-rules.pro")
+    }
+  }
+
+  // kotlin_builtins duplication between 1.5.20 and 1.6.10?
+  packagingOptions.exclude("**/*.kotlin_*")
+
+  targetProjectPath = ":benchmarks:performance-poetry:complex-poetry"
+  experimentalProperties["android.experimental.self-instrumenting"] = true
+}
+
+dependencies {
+  implementation(project(":benchmarks:performance-poetry:complex-poetry"))
```

Comment:
so what's a complex benchmark? What's a not complex one? Are there more interesting words we can use to describe the difference?

Reply:
Unless... this is about "ComplexPoetry". Why couldn't that have been just Poetry?!

Reply:
lol. This is a 'Complex' version of the Poetry app.